docs/get-started/using-standalone.md

```diff
@@ -33,7 +33,7 @@ const deckgl = new Deck({
       data: [
         {position: [-122.45, 37.8], color: [255, 0, 0], radius: 100}
       ],
-      getColor: d => d.color,
+      getFillColor: d => d.color,
```

Comment:
If you use "getColor" instead of "getFillColor", it will throw a warning:
"deck: `ScatterplotLayer: getColor` is deprecated and will be removed in a later version. Use `getFillColor/getLineColor` instead"